programs/hydra/src/processors/init/transfer_to_hodling.rs

```diff
@@ -0,0 +1,42 @@
+use crate::error::{OrArithError};
+use crate::state::{Fanout};
+use anchor_lang::prelude::*;
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Default)]
+pub struct TransferToArgs {
+    pub bump_seed: u8,
+    pub native_account_bump_seed: u8,
+}
+
+#[derive(Accounts)]
+#[instruction(args: TransferToArgs)]
+pub struct TransferToHodling<'info> {
+    #[account(
+        mut,
+        seeds = [b"fanout-config", fanout.name.as_bytes()],
+        bump = fanout.bump_seed,
+    )]
+    pub fanout: Box<Account<'info, Fanout>>,
+    #[account(mut)]
+    /// CHECK: Could be native or Token Account
+    pub holding_account: UncheckedAccount<'info>,
+    pub system_program: Program<'info, System>,
+    pub rent: Sysvar<'info, Rent>,
+}
+pub fn transfer_to_hodling(
+    ctx: Context<TransferToHodling>,
+    _args: TransferToArgs
+) -> Result<()> {
+    let fanout = &mut ctx.accounts.fanout.to_account_info();
+    let hodling = ctx.accounts.holding_account.to_account_info();
+    let snapshot: u64 = fanout.lamports();
+
+    **fanout.lamports.borrow_mut() = 0;
```

Comment:
This is going to drop the fanout to below rent-exempt minimum?

Reply:
afaik from my previous work with candy machine refunds it will empty the thing and solana conveniently kicks it out (can't afford rent).

It's ? a weird process?

Reply:
oh - good point - we don't want to destroy the fanout in this particular case. 

There's a function somewhere for RENT_EXEMPT_MINIMUM or w/e?

Reply:
https://github.com/cardinal-labs/cardinal-token-manager/blob/3825e0eb70007a2b1e263dedeaf02b693c9614b7/programs/cardinal-token-manager/src/instructions/invalidate.rs#L199-L205
```
            let required_lamports = ctx.accounts.rent.minimum_balance(token_manager.to_account_info().data_len());
            let token_manager_lamports = token_manager.to_account_info().lamports();
            if token_manager_lamports > required_lamports {
                let diff = token_manager_lamports.checked_sub(required_lamports).expect("Sub error");
                **token_manager.to_account_info().try_borrow_mut_lamports()? = required_lamports;
                **ctx.accounts.collector.to_account_info().try_borrow_mut_lamports()? = ctx.accounts.collector.to_account_info().lamports().checked_add(diff).expect("Add error");
            };

```

this example gets rent minimum and removes any extra lamports above that